### PR TITLE
3661/Fixing docu

### PR DESCRIPTION
**Pull request by jona-samuel**

### CODE REVIEW

File: doc/policies/authentication.rst [hunk 1]
```diff
@@ -782,7 +782,7 @@ type: string
 
 This action sets a list of the client mode in the preferred order. You can enter the different client
 modes in the order you like. For example: "interactive, webauthn, poll, u2f". The client you're using
-will show you the riad login for you're preferred client mode. For example: this is you're list:
-"interactive, webauthn, poll, u2f" and in you're multi challenge are a webauthn and u2f token,
-than you're client will automatically show you the login for a webauthn token.
+will show you the correct login for your preferred client mode. For example, this is your list:
+"interactive, webauthn, poll, u2f" and in your multi challenge are a webauthn and u2f token,
+than your client will automatically show you the login for a webauthn token.
```
Comment (plettich) [inline doc/policies/authentication.rst:787]: ... then Your client ...

File: doc/policies/authentication.rst [hunk 2]
```diff
@@ -782,7 +782,7 @@ type: string
 
 This action sets a list of the client mode in the preferred order. You can enter the different client
 modes in the order you like. For example: "interactive, webauthn, poll, u2f". The client you're using
-will show you the riad login for you're preferred client mode. For example: this is you're list:
-"interactive, webauthn, poll, u2f" and in you're multi challenge are a webauthn and u2f token,
-than you're client will automatically show you the login for a webauthn token.
+will show you the correct login for your preferred client mode. For example, this is your list:
+"interactive, webauthn, poll, u2f" and in your multi challenge are a webauthn and u2f token,
```
Comment (plettich) [inline doc/policies/authentication.rst:786]: ... in Your multi-challenge list are ...

File: doc/policies/authentication.rst [hunk 3]
```diff
@@ -782,7 +782,7 @@ type: string
...
+than your client will automatically show you the login for a webauthn token.
  .. note:: The default list is "interactive, webauthn, poll, u2f"
```
Comment (plettich) [inline doc/policies/authentication.rst:788]: please also remove the commas here.

**Review by plettich — CHANGES_REQUESTED** [review]

File: doc/policies/authentication.rst (hunk 3 above)
Comment (jona-samuel) [reply doc/policies/authentication.rst]: https://github.com/privacyidea/privacyidea/pull/3665/commits/7bf9b89d96e95b579eda0b912700731d23ba580a

File: doc/policies/authentication.rst (hunk 2 above)
Comment (jona-samuel) [reply doc/policies/authentication.rst]: https://github.com/privacyidea/privacyidea/pull/3665/commits/7bf9b89d96e95b579eda0b912700731d23ba580a

File: doc/policies/authentication.rst (hunk 1 above)
Comment (jona-samuel) [reply doc/policies/authentication.rst]: https://github.com/privacyidea/privacyidea/pull/3665/commits/7bf9b89d96e95b579eda0b912700731d23ba580a

File: doc/policies/authentication.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (jona-samuel) [reply doc/policies/authentication.rst]: https://github.com/privacyidea/privacyidea/pull/3665/commits/7bf9b89d96e95b579eda0b912700731d23ba580a

**Review by plettich — APPROVED** [review]

**Pull request merged**